PackageConfig.cmake.in

```diff
@@ -0,0 +1,16 @@
+@PACKAGE_INIT@
+
+#@PROJECT_NAME@-config.cmake
+#
+# Imported interface targets provided:
+#  * @PROJECT_NAME@::@PROJECT_NAME@_4 - real32 library target
+#  * @PROJECT_NAME@::@PROJECT_NAME@_8 - real64 library target
+#  * @PROJECT_NAME@::@PROJECT_NAME@_d - mixed library target
+
+# Include targets file.  This will create IMPORTED target @PROJECT_NAME@
+include("${CMAKE_CURRENT_LIST_DIR}/@PROJECT_NAME@-targets.cmake")
+
```

Comment:
we should add the dependencies here.

Reply:
MPI is a dependency too.

Reply:
Or is MPI resolved in nemsio. I didn't see a find_dependency(MPI) in NEMSIO packageconfig.

Reply:
I will do that

Reply:
I am in two minds about this. Definitely add in NEMSIO. But, since we are explicitly linking with MPI here, we should add both the find_package (in CMakelists) and find_dependency(in packageconfig) here.
If you don't want to add it here, we should remove the reference to MPI::MPI_Fortran in target_link_libraries, because that will come from NEMSIO transitive properties (or so I hope).

Reply:
Right, I agree. Forgot to remove where we link to it. w3mc does not reference MPI directly and should just rely on finding it through nemsio.